### ISSUE TITLE
Request specification clarification on sequence truncations

### ISSUE THREAD

**Issue by dtabb73** [HUPO-PSI/ProForma#4]
I would like to see a paragraph in the specification indicating how proteoform sequence truncations are to be specified.  N-terminal truncations may be biological, as in the removal of the initial Met (perhaps with PTM) or the cleavage of a signal peptide or the action of a viral protease.  The truncations may be instead be related to sample treatment, such as a rare cutter like CNBr for middle-down proteomics or due to a "hot" ion source.  I believe ProForma should specify how a proteoform sequence compares to the sequence described by the accession, such as indicating the position of the first and last amino acids in the accession's sequence.  Are amino acids preceding and succeeding the proteoform sequence expected to be included?

**Comment by javizca** [comment]
In my view, this is "metadata" information on top of the actual protein sequence. In the current version of the specification, we decided to handle those issues using the INFO tag providing the metadata there as free text. 

Standardise every single annotation at this point is unfeasible in my view.

**Comment by edeutsch** [comment]
I think all this is beyond the scope of ProForma 2.0. ProForma is designed to describe the molecule that (someone claims) yielded a spectrum. Information about:
- what is the parent accession
- where in the parent is the peptidoform
- what is missing in the peptidoform that might be expected based on the assumed parent
- what are preceding and following amino acids
is not designed to be encoded in ProForma 2.0. There could be 20 parent accessions, each with different offset, preceding and following amino acids. Try to capture that in ProForma 2.0 would be hideous.

Basically ProForma is about what you think you have observed, not about what you infer about context of that observation.
(with a minor exception that there is some ambiguity between I/L and a few other isobaric ambiguiities where ProForma allows the user to express one, but it is implied that isobaric alternatives are possible)

**Comment by douweschulte** [comment]
Although the specific use case of sequence truncation/parentage does not sit well in the current design and goals of ProForma with version 2.1 a new feature is introduced that can be used for such purposes. This is the peptidoform name `(>Identifier)PEPTIDE` which can be used similarly to a fasta identifier to encode any tool specific information.

**Comment by douweschulte** [comment]
A new proposal by Patrick Garrett (@pgarrett-scripps) aims to allow encoding this sort of information: https://docs.google.com/document/d/1HaSOMNmdFCxy0G6lq1Yci03z-4esmQoe_hSyoqTeRnc/edit?tab=t.0#heading=h.h395hgv0u0zk (suggestions can be left in that file as well).
